.github/workflows/tests-unit.yml

```diff
@@ -7,19 +7,15 @@ jobs:
     name: Core Unit Tests
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - uses: actions/checkout@v2
-    - name: Cache node modules
-      uses: actions/cache@v2
-      with:
-        path: node_modules
-        key: yarn-2-cache-v1-${{ hashFiles('**/yarn.lock') }}
-    - name: install, bootstrap
-      run: |
-        yarn install --immutable
-        yarn bootstrap --core
-    - name: test
-      run: |
-        yarn test --runInBand --ci
+      - uses: actions/setup-node@v1
```

Comment:
Considering it was added in 2.2.0 you would need to update this to v2

Reply:
Yo are totally right

Reply:
https://github.com/storybookjs/storybook/pull/15523